extensions/processing/README.md

```diff
@@ -0,0 +1,64 @@
+# Processing Extension Specification
+
+- **Title: Processing**
+- **Identifier: proc**
+- **Field Name Prefix: proc**
+- **Scope: Item, Collection**
+- **Extension [Maturity Classification](../README.md#extension-maturity): Proposal**
+
+Processing metadata is considered to be data that indicate from which processing chain a data originates and how the data itself has been produced. Overall, it helps to increase traceability and search among processing levels and multiple algorithm versions.
+
+Often, data items are the result of one or more waterfall processing pipeline. Tracing information such as the processing facility , the algorithm version or the processing date helps in the data version management.
+
+This extension applies to STAC Items and STAC Collections. As these processing information are often closely bound to the collection level and therefore are shared across all items, it is recommended adding the fields to the corresponding STAC Collection.
+
+- [Examples](examples/): [Sentinel-1 Item](examples/sentinel1-item.json),  [Sentinel-2 Level 1C Collection](examples/sentinel2-collection.json)
+- [JSON Schema](json-schema/schema.json)
+
+## Item and Collection fields
+
+For Items, the fields are placed in the properties. For Collections, the fields are placed in the summaries of the Collection.
+
+| Field Name                  | Type      | Description                                                  |
+| --------------------------- | --------- | ------------------------------------------------------------ |
+| proc:name         | string | The name of the processing of the data that is commonly used. For example, `GRD Post Processing` for "GRD" product of Sentinel-1 satellites. |
+| proc:level        | string | **REQUIRED.** The name commonly used to refer to the processing level to make it easier to search for product level across collections or items. The short name must be used (only `L`, not `Level`). See the [list of suggested processing levels](#suggested-processing-levels). |
+| proc:facility     | string    | The name of the facility that produced the data. For example, `Copernicus S1 Core Ground Segment - DPA` for product of Sentinel-1 satellites. |
```

Comment:
What is the use case for this field?

Reply:
If you mean the facility, it allows filtering product on their processing center origin. For instance, to select products produced by a specific ground segment components.

Reply:
Okay, so that's mainly for the "original" products, I guess? I must admit I wouldn't know what to put there if I process something at - for example - AWS or even locally. ;-)